articles/managing-heap/index.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: engineering-education
+status: publish
+published: true
+slug: managing-heap
+title: Managing the Heap in C
+description: Working with the heap in C is very difficult. Even Google struggles with it. But it's very important to not have memory issues in your code.
+author: mike-white
+date: 2020-07-16T00:00:00-07:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/managing-heap/hero.jpg
+    alt: memory drive image example
+---
+Memory management is very important. In Google Chrome, [70% of security bugs are memory problems](https://zdnet.com/article/chrom-70-of-all-security-bugs-are-memory-safety-issues/). Fixing memory management would lead to much more reliable and secure programs. There are two possible solutions to this. You can use a language with [better](https://www.rust-lang.org) or [easier](https://www.java.com), or just learn how to use memory properly. We will talk about the latter today.
```

Comment:
Link to https://zdnet.com/article/chrom-70-of-all-security-bugs-are-memory-safety-issues/ returns a 404

Reply:
> You can use a language with better or easier,

Feels like this sentence should read more along the lines of 
"You can use a language that manages memory better or easier"

Reply:
I think I forgot an 'e' in the link. It seems to work now.

I think I just accidentally deleted the words, "memory management".